tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
The original test should be correct because a few line above there `$url->setRootControllerNamespace('namespace');` which set the root namespace to `namespace`, and due to this `something` namespace would include the leading backslash.